_config/products.yml

```diff
@@ -728,13 +745,14 @@ jbt_core:
             md5_url: http://sourceforge.net/projects/jboss/files/JBossTools/jbosstools4.3.0.x/jbosstools-4.3.0.Alpha1_2015-02-15_14-44-55-B13-browsersim-standalone.zip.MD5
           - name: Release Notes
             url: http://tools.jboss.org/documentation/whatsnew/index.html
-      4.3.0.Nightly:
-        update_site_url: http://download.jboss.org/jbosstools/updates/nightly/mars/
+      4.3.x.Nightly:
+        update_site_url: http://download.jboss.org/jbosstools/mars/snapshots/updates/
```

Comment:
Actually, no. http://download.jboss.org/jbosstools/mars/snapshots/updates/ contains the same bits as  http://download.jboss.org/jbosstools/updates/nightly/mars/ but follows the new path convention we started for JBDS 9.